samples/client/petstore/java/native-async/src/main/java/org/openapitools/client/api/PetApi.java

```diff
@@ -165,6 +166,7 @@ private String formatExceptionMessage(String operationId, int statusCode, String
    * 
    * @param petId Pet id to delete (required)
    * @param apiKey  (optional)
+   * @return CompletableFuture&lt;Void&gt;
```

Comment:
What about using `{{{...}}}` (original value) instead of `{{..}}` (HTML escaped)?

Reply:
Unfortunately I do not fully understand requested change. Can you please describe more in detail? 
We do not have value in hand and I assume in the end we want to have `&lt;` and not the `<`.

Reply:
Sorry. I was confused with C# doc in the code. The change looks good. Thanks for the PR.